package.json

```diff
@@ -16,6 +16,6 @@
     "changelog:api": "cd api && yarn changelog",
     "changelog:app": "cd app && yarn changelog",
     "changelog:extension": "cd extension && yarn changelog",
-    "lint:global": "cd api && yarn lint --fix && cd .. && cd app && yarn lint --fix && cd .. && cd extension && yarn lint:fix && cd .."
+    "lint:lunie": "cd api && yarn lint --fix && cd .. && cd app && yarn lint --fix && cd .."
```

Comment:
🙈

Reply:
why this change? at least leave it global?

Reply:
because usually we work with API and Lunie Core and it takes a good amount of time linting extension additionally

We can have both of them actually